client/components/backup-storage-space/storage-usage-help-popover.tsx

```diff
@@ -0,0 +1,101 @@
+import { Button, Gridicon, Popover } from '@automattic/components';
+import { useTranslate } from 'i18n-calypso';
+import * as React from 'react';
+
+type OwnProps = {
+	className?: string;
+	storageUpgradeUrl: string;
+	forecastInDays: number | undefined;
+	onClickedPurchase: () => void;
+};
+const StorageHelpPopover: React.FC< OwnProps > = ( {
+	className,
+	storageUpgradeUrl,
+	forecastInDays,
+	onClickedPurchase,
+} ) => {
+	const translate = useTranslate();
+	const [ isPopoverVisible, setPopoverVisible ] = React.useState< boolean >( true );
```

Comment:
> Can we trigger them initially, so they’re visible and then the info icon stays there permanently?

Ah, forgot to mention it in the PR, yes, as per [above comment](p1HpG7-lPI-p2#comment-62476) in i1 of the design, I made it to open by default.

@keoshi Could you confirm this is the behavior you wanted

Note: Maybe we can show it the first time, and if dismissed it will remain dismissed by default and can be re-opened from the info icon (using local storage)

Reply:
> Maybe we can show it the first time, and if dismissed it will remain dismissed by default and can be re-opened from the info icon (using local storage)

Exactly this! It should:
- Be already open on first visit if we forecast their site will go over the limit.
- Be dismissible.
- Be re-opened by clicking/tapping the info icon.

Thank you, @rustamshaikh !

Reply:
@Initsogar @keoshi I handled the case so that the popover will be pre-opened until the user dismisses it one time to ensure the user notices it at least once. Once dismissed, it can be re-opened from the info icon, but it will be no longer pre-opened for the user on page reload going further.

Reply:
Makes sense to me 👍🏽

Reply:
Thanks @Initsogar & @keoshi :)